examples/va/hci/edpm-pre-ceph/values.yaml

```diff
@@ -28,8 +28,12 @@ data:
         edpm_logrotate_crond_image: '{{ registry_url }}/openstack-cron:{{ image_tag }}'
         edpm_network_config_hide_sensitive_logs: false
         edpm_network_config_os_net_config_mappings:
-          edpm-compute:
-            nic2: enp7s0
+          edpm-compute-0:
+            nic2: 6a:fe:54:3f:8a:02 # CHANGEME
+          edpm-compute-1:
+            nic2: 6b:fe:54:3f:8a:02 # CHANGEME
+          edpm-compute-2:
+            nic2: 6c:fe:54:3f:8a:02 # CHANGEME
```

Comment:
I talked to Andrew and I understand better what is happening here.

* Lines 31 and 32 are not doing anything and `os-net-config` was ignoring them and just using the defaults
* The defaults worked in a lot of testing but fail in some cases (which this PR is trying to fix with an example)
* The defaults can be corrected by using the newly proposed syntax, provided the mac addressees are correct
* The alternative to using mac addresses is to provide DMI but what would be more difficult for ci-framework to provide in the values.yaml file it will replace when it overrides this example

So I'm fine with this example being changed since I now understand why.

I expect ci-framework has the desired mac address stored as a fact so maybe it could generate a values.yaml like this example.